bindata/network/egress-router/000-nad.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: egress-router
+spec:
+  config: '{
+    "cniVersion": "0.4.0",
+    "type": "egress-router",
+    "name": "egress-router-config",
+    "ip": {
+      "addresses": [
+        "192.168.10.99/24"
```

Comment:
This is just for testing, right?

Reply:
Yep, I'll change that for a template in the next iteration